docs/concepts/weak-subjectivity.md

```diff
@@ -6,31 +6,62 @@ sidebar_position: 7
 
-For example, if 1/3 of validators withdraw their stake and continue signing blocks and attestations, they can form a chain which conflicts with the finalized state. If your node is far enough behind the chain head not to be aware that they've withdrawn their funds, these validators can act dishonestly and continue feeding you blocks to lead you down the wrong chain.
+For example, if 1/3 of validators withdraw their stake and continue signing blocks and attestations,
+they can form a chain which conflicts with the finalized state.
+If your node is far enough behind the chain head not to be aware that they've withdrawn their funds,
+these validators can act dishonestly and continue feeding you blocks to lead you down the wrong chain.
 
 :::note
-
 If a node is aware that a validator has withdrawn its funds, the node will reject the validator's attestations.
-
 :::
 
-## Safely sync your node
+At a high-level, the weak subjectivity period is the period of time that a node can be behind the
+chain and trust that it is following the correct chain.
+In practice, the weak subjectivity mechanism tells Teku if the latest checkpoint is too old to
+continue syncing from it (either when starting a new node or after your node has been offline for a while).
 
-Teku provides two methods to safely sync a node that is new to the network or has been offline for an extended period.
 
-1. Use [`--initial-state`](../reference/cli/index.md#initial-state) to supply an SSZ encoded state file from which to sync.
-2. Use [`--ws-checkpoint`](../reference/cli/index.md#ws-checkpoint) to supply a weak subjectivity checkpoint by which a node can securely validate its view of the current state.
+## Safely sync your node
 
-We recommend using `--initial-state`. It provides the same security benefits as `--ws-checkpoint`, but syncs faster. The only exception is when syncing an archive node, in which case, use `--ws-checkpoint`.
+Teku provides three methods to safely sync a node that is new to the network or has been offline for an extended period.
 
-:::tip
+1. Use [`--checkpoint-sync-url`](../reference/cli/index.md#checkpoint-sync-url) to supply a URL of a
+    checkpoint state endpoint from which to sync.
+2. Use [`--initial-state`](../reference/cli/index.md#initial-state) to supply an SSZ encoded state
+    file from which to sync.
+3. Use [`--ws-checkpoint`](../reference/cli/index.md#ws-checkpoint) to supply a weak subjectivity
+    checkpoint by which a node can securely validate its view of the current state.
 
-Use the [`/eth/v2/debug/beacon/states/<state_id>`](https://consensys.github.io/teku/#operation/getEthV2DebugBeaconStatesWithState_id) API on an updated node to download a recent finalized state as an SSZ encoded state file.
+We recommend using `--checkpoint-sync-url` or `--initial-state`.
+They provide the same security benefits as `--ws-checkpoint`, but sync faster.
+The only exception is when syncing an archive node, in which case, use `--ws-checkpoint`.
```

Comment:
I think we might need to revisit this recommendation. For mainnet, we should only recommend people use `checkpoint-sync-url`.